configure.in

```diff
@@ -136,76 +128,72 @@ if test "$enable_openssl" = "yes"; then
              [AC_MSG_FAILURE([can't find openssl >1.0.1 crypto lib])])
```

Comment:
This isn't new in this version of the code -- you're just fixing the quoting -- but does this really require >1.0.1, as opposed to >= 1.0.1?